src/test/tests.ts

```diff
@@ -133,22 +127,22 @@ const subscriptionManager = new SubscriptionManager({
 
 // indirect call to support spying
 const handlers = {
-  onSubscribe: (msg: SubscribeMessage, params: SubscriptionOptions, webSocketRequest: WebSocket) => {
-    return Promise.resolve(Object.assign({}, params, {context: msg['context']}));
+  onSubscribe: (msg: OperationMessage, params: SubscriptionOptions, webSocketRequest: WebSocket) => {
```

Comment:
hmm, for unsubscribe we need to hook it and flat it like it was before,
the idea is that people that are using unsubsribe will get the old behavior,
but onOperation is the new one